core/frontend/src/render/primitives/geometry/GeometryAccumulator.ts

```diff
@@ -60,7 +60,7 @@ export class GeometryAccumulator {
   }
 
   private calculateTransform(transform: Transform, range: Range3d): Transform {
-    const xform = transform.clone();
+    const xform = this.haveTransform ? transform : transform.clone();
```

Comment:
Sorry to nitpick but
```
private calculateTransform(transform: Transform, range: Range3d): Transform {
  if (this.haveTransform)
    transform = this._transform.multiplyTransformTransform(transform);

  transform.multiplyRange(range, range);
  return transform;
}
```

Reply:
All good. That's a better usage of the API; thanks.